common/bitstream.h

```diff
@@ -33,26 +33,35 @@
 
 namespace Common {
 
+/**
+ * @defgroup common_bitstream Bit stream
+ * @ingroup common
+ *
+ * @brief  API for implementing a bit stream.
+ *
+ * @{
+ */
+
 /**
  * A template implementing a bit stream for different data memory layouts.
  *
  * Such a bit stream reads valueBits-wide values from the data stream and
  * gives access to their bits, one at a time.
  *
  * For example, a bit stream with the layout parameters 32, true, false
- * for valueBits, isLE and isMSB2LSB, reads 32bit little-endian values
+ * for valueBits, isLE and isMSB2LSB, reads 32-bit little-endian values
  * from the data stream and hands out the bits in the order of LSB to MSB.
  */
 template<class STREAM, int valueBits, bool isLE, bool MSB2LSB>
```

Comment:
This class is nicely documented, only misses docs for two small structures in lines 277 and 281.